src/background/services/sessionService.ts

```diff
@@ -92,10 +92,20 @@ export class SessionService {
   ): Promise<LoginSession> {
     request.timeout = request.timeout ?? DEFAULT_REQUEST_TIMEOUT
     request.secureLogin = request.secureLogin ?? 'disabled'
+    request.reusePolicy = request.reusePolicy ?? 'disabled'
     request.from = request.from ?? 'any'
 
-    if (!request.authMethods || request.authMethods.length === 0)
+    if (!request.authMethods || request.authMethods.length === 0) {
       throw new Error(`"authMethods" is required.`)
+    }
+
+    if (!['disabled', 'optional', 'required'].includes(request.secureLogin)) {
```

Comment:
Here is a hard code. If there will be new variants, changes in LoginRequest type, the error would be here.
The solution may be something like this:
```ts
enum SecureLogin {
  'required',
  'optional',
  'disabled'
}

type LoginRequest = {
  // ...
  secureLogin: SecureLogin
}

const request = {
 //...
  secureLogin: 'optional'
}

console.log(SecureLogin[request.secureLogin])

if (SecureLogin[request.secureLogin] === undefined) {
  console.log('Invalid "secureLogin" value.')
} else {
  console.log('Cool!')
}
```